ios/utils/RCTSurfaceTouchHandler+RNSUtility.mm

```diff
@@ -0,0 +1,14 @@
+#ifdef RCT_NEW_ARCH_ENABLED
+#import "RCTSurfaceTouchHandler+RNSUtility.h"
+
+@implementation RCTSurfaceTouchHandler (RNSUtility)
+
+- (void)rns_cancelTouches
+{
+  [self setEnabled:NO];
+  [self setEnabled:YES];
```

Comment:
I don't quite like it. Can't we call `touchesCancelled` or `touchesEnded` method to somehow cancel the touch in other way?

Reply:
I think changing this code is out of the scope of this PR. I'm reusing already existing & **working** code present in `RNSScreenStack.mm` & this change is only a refactor to improve code readability.

Reply:
Also it is worth mentioned that this is approach used in RN internals